py/desispec/data/qa/qa-params.yaml

```diff
@@ -0,0 +1,86 @@
+# QA parameters read by exposure_qa.py
+exposure_qa:
+
+ # cutoff fiber positioning offset in mm
+ max_fiber_offset_mm : 0.03
+
+ # maximum read noise rms per CCD pixel in electrons
+ max_readnoise : 10
+
+ # maximum fraction of fibers with bad positioning
+ # per petal
+ max_frac_of_bad_positions_per_petal : 0.5
+
+ # minimum number of valid standard stars per petal
+ min_number_of_good_stdstars_per_petal : 3
+
+ # maximum rms of calibration error per star
+ # (ratio of calibrated flux to model in rband)
+ max_rms_of_rflux_ratio_of_stdstars : 0.2
+
+ # tsnr parameters
+ tsnr2_band : r
+ tsnr2_key : TSNR2_LRG_R
+ tsnr2_petal_minfrac : 0.5
+ tsnr2_petal_maxfrac : 2.
+ tsnr2_for_efftime_dark : 'TSNR2_ELG'
+ tsnr2_for_efftime_bright : 'TSNR2_BGS'
+ tsnr2_for_efftime_backup : 'TSNR2_BGS'
+
+ # bad POSITIONING flag
+ bad_positions_mask : "STUCKPOSITIONER|BROKENFIBER|RESTRICTED|MISSINGPOSITION|BADPOSITION|POORPOSITION"
+ # bad QAFIBERSTATUS flag
+ bad_qafstatus_mask : "UNASSIGNED|STUCKPOSITIONER|BROKENFIBER|RESTRICTED|MISSINGPOSITION|BADPOSITION|POORPOSITION|BADFIBER|BADTRACE|BADFLAT|BADARC|MANYBADCOL|MANYREJECTED|BADAMPB|BADAMPR|BADAMPZ|BADPETALPOS|BADPETALSKY|BADPETALSTDSTAR|BADPETALFLUXCAL|BADPETALSNR|BADREADNOISE"
```

Comment:
For both `bad_positions_mask` and `bad_qafstatus_mask`, I don't think STUCKPOSITIONER and RESTRICTED should count as bad, since they could be validly used for SKY targets (or even science targets for RESTRICTED positioners).  If they couldn't be used for that it will appear through UNASSIGNED, and if they aren't at the expected/requested location it will show up through POORPOSITION, BADPOSITION regardless of whether they are RESTRICTED or STUCK.

Reply:
ok. good point.